futures-util/Cargo.toml

```diff
@@ -17,19 +17,23 @@ name = "futures_util"
 [features]
 std = ["alloc", "futures-core-preview/std", "futures-io-preview/std", "futures-sink-preview/std", "slab", "memchr"]
 default = ["std"]
-async-await = ["std", "futures-select-macro-preview", "proc-macro-hack", "proc-macro-nested", "rand"]
+async-await = ["std"]
 compat = ["std", "futures_01"]
-io-compat = ["compat", "tokio-io"]
+io-compat = ["io", "compat", "tokio-io"]
 bench = []
 nightly = ["futures-core-preview/nightly", "futures-sink-preview/nightly"]
 cfg-target-has-atomic = ["futures-core-preview/cfg-target-has-atomic"]
 alloc = ["futures-core-preview/alloc", "futures-sink-preview/alloc"]
+sink = ["futures-sink-preview", "futures-channel-preview/sink"]
```

Comment:
I would like to depend on futures-util's sink helpers w/o pulling in `futures-channel`.

Reply:
Oh, that makes sense. Fixed.